src/main/scala/org/broadinstitute/dsde/workbench/avram/api/AvramRoutes.scala

```diff
@@ -27,33 +31,45 @@ class AvramRoutes {
 
   val configFactory = ConfigFactory.parseResources("app.conf").withFallback(ConfigFactory.load())
   private val dbcpDataSourceConfig = configFactory.as[DbcpDataSourceConfig]("dbcpDataSource")
-  val dataSource = new DataSourceFactory(dbcpDataSourceConfig)
-
+  val dataReference =  DbReference(dbcpDataSourceConfig)
 
   @ApiMethod(name = "ping", httpMethod = "get", path = "ping")
   def ping: Pong = {
     Pong()
   }
 
-  // TODO: remove this endpoint when we have more meaningful ways to test database queries
-  @ApiMethod(name = "now", httpMethod = "get", path = "now")
-  def now: Now = {
-    // Explicitly use a Future to make sure the implicit ExecutionContext is being used
-    Now(Await.result(Future(fetchTimestampFromDBWithSlick()), Duration.Inf))
+  //I'm going to delete this obvs but here's how we might use a Transaction
+  @ApiMethod(name = "addCollection", httpMethod = "get", path = "addCollection")
+  def addCollection: Collection = {
+    Await.result(
+      for {
+        _ <- dataReference.inTransaction { dataAccess => dataAccess.collectionQuery.save("testResource", "samResource1") }
+      } yield {
+        Collection("testResource", "samResource2")
+      }, Duration.Inf)
```

Comment:
This is going to be deleted, just wanted to show the transaction. I've tested adding entities with JSONB and it works.